include/psp2kern/io/devctl.h

```diff
@@ -0,0 +1,39 @@
+/**
+ * \kernelgroup{SceDevCtl}
+ * \usage{psp2kern/io/devctl.h,?}
```

Comment:
Please change `?` to `-lstub_name`.
The `?` left in the include files are there out of laziness, it's not an example to follow.